docs/tutorial/batch-minting.md

```diff
@@ -0,0 +1,246 @@
+---
+title: Batch-minting NFTs on Ethereum 🚧
+description: This tutorial addresses some techniques in batch-minting big numbers of Ethereum non-fungible tokens (NFTs) as well as common idioms of uploading files and metadata on to IPFS/Filecoin via nft.storage.
+issueUrl: https://github.com/protocol/nft-website/issues/253
+related: 'Lazy minting': https://nftschool.dev/tutorial/lazy-minting/
+---
+ # Batch-minting NFTs on Ethereum
+
+ > TODO: Add missing links
+
+ This tutorial addresses some techniques in batch-minting big numbers of Ethereum non-fungible tokens (NFTs) as well as common idioms of uploading files and metadata on to IPFS/Filecoin via nft.storage.
+
+See also: [Lazy minting](./lazy-minting.md)
+
+Quite often,  NFT projects consist of hundreds if not thousands of files and metadata starting their lives out on a computer file system, waiting nervously to be part of a mint.
+
+There are many cases in which instead of minting NFTs just in time (or lazily) once the buyer has concluded a purchase, it is desirable to mint several all at once. Some common ones are:
+
+- Event tickets
+- Packs
+- Game items
+
+In all these scenarios, NFTs are being treated closer to commodities than in the case in which collectors collect them as discrete artworks. Some challenges that lie in minting a batch of many tokens include exceedingly high gas fees on the Ethereum network, atomically synchronizing between mint and upload failures, and performance of uploading files to nft.storage, etc.
+
+## Pre-processing and project structure
+
+> TODO: Add content here
+
+## Storage
+
```

Comment:
yeah - i'd outline the tradeoffs with going over iteratively (harder to manage all the CIDs, but potentially more resilient because you can retry individual uploads) vs. directory (easier to manage, but higher risk of something going wrong when you upload a large directory; also doing programmatically means you need to fit the whole directory in-memory, though you could use NFT Up to get around this)

I'd also mention that there's a new upload client coming in the coming weeks that will allow you to upload without memory constraints and more resiliency!